Makefile.env.mk

```diff
@@ -40,7 +40,7 @@ ALERTMANAGER_IMAGE ?= prom/alertmanager:v0.15.2
 GRAFANA_IMAGE ?= grafana/grafana:5.3.1
 APPLICATION_MONITORING_OPERATOR_IMAGE ?= quay.io/integreatly/application-monitoring-operator:0.0.27
 KUBE_STATE_METRICS_IMAGE ?= quay.io/coreos/kube-state-metrics:v1.4.0
-BROKER_IMAGE ?= quay.io/enmasse/artemis-base:2.10.1
+BROKER_IMAGE ?= quay.io/enmasse/artemis-base:5a47cee4
```

Comment:
Is using a SHA here just a temporary measure?

Reply:
Yes, temporary until the base image PR is merged (but using this hash now so that i'm able to run tests).